solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -40,6 +33,10 @@ Following parameter/s are passed when the cluster deployment command executes. I
 | region | ap-south-1 | You can pick any region from this region code : https://awsregion.info/  |
 | security_group_cidr | 134.204.222.36/32  | You need to find the own Public IP using this command : `curl ipinfo.io/ip`. Also calculate CIDR for IP using Subnet Calculator from https://mxtoolbox.com/subnetcalculator.aspx |
 | key_name | devops-key | You can pass .pem key file name to login to aws EC2 instance in `key_name`. |
+| ebs_volume_count | 9 |  You can select the number of EBS volumes |
+| ebs_volume_size | 10 |  You can select the EBS volume size |
+| instance_count | 3  | You can select the number of EC2 instances |
```

Comment:
In order to perform 3 node deployment, we need to have 1 node to perform build/ so this has to be 4 instances

Reply:
No, this will be done on 3 nodes as local host is not counted so count is 3